lua/fishing_mod/cl_shop_menu.lua

```diff
@@ -23,74 +128,153 @@ function PANEL:Init()
 	
 	self.upgrade = vgui.Create("Fishingmod:Upgrade", self)
 
+	self.customization = vgui.Create("Fishingmod:Customization", self)
+
 	local xpx, xpy = self:GetSize()
 
 	local upgradesbutton = vgui.Create("DButton", self) -- upgrades
 	local baitsbutton = vgui.Create("DButton", self) -- baits shop
+	local customizationbutton = vgui.Create("DButton", self) -- customization tab
+
 	upgradesbutton.selected = true
 	baitsbutton.selected = false
 	upgradesbutton:SetPos(3, 24)
-	upgradesbutton:SetSize((xpx - 6) / 2, 22)
+	upgradesbutton:SetSize((xpx - 6) / 3, 22)
 	upgradesbutton:SetText("Upgrades")
 	function upgradesbutton.Think()
 		xpx, xpy = self:GetSize()
-		upgradesbutton:SetSize((xpx - 6) / 2, 22)
+		upgradesbutton:SetSize((xpx - 6) / 3, 22)
 	end
-	upgradesbutton:SetTextColor(selected)
+	upgradesbutton:SetTextColor(ui_text)
 	upgradesbutton.DoClick = function()
-		baitsbutton:SetColor(grey)
-		upgradesbutton:SetColor(selected)
+		upgradesbutton:SetColor(ui_text)
+		baitsbutton:SetColor(nopres)
+		customizationbutton:SetColor(nopres)
+
+		baitsbutton:SetTextColor(button_not_selected)
+		customizationbutton:SetTextColor(button_not_selected)
+
 		upgradesbutton.selected = true
 		baitsbutton.selected = false
+		customizationbutton.selected = false
+
+		self.customization:Hide()
 		self.upgrade:Show()
 		self.baitshop:Hide()
 	end
 	upgradesbutton.Paint = function(self, w, h)
-		if(upgradesbutton:IsHovered()) then
-			surface.SetDrawColor(0, 0, 0, 144)
-		elseif(upgradesbutton.selected) then
-			surface.SetDrawColor(0, 0, 0, 144)
+		if self.selected then
+			upgradesbutton:SetColor(ui_text)
+		elseif not self.selected then
+			upgradesbutton:SetColor(button_not_selected)
+		end
+		if upgradesbutton:IsHovered() then
+			surface.SetDrawColor(back_ground.r, back_ground.g, back_ground.b, back_ground.a)
+		elseif upgradesbutton.selected then
+			surface.SetDrawColor(back_ground.r, back_ground.g, back_ground.b, back_ground.a)
 		else
-			surface.SetDrawColor(0, 0, 0, 72)
+			surface.SetDrawColor(nopres.r, nopres.g, nopres.b, nopres.a)
 		end
 		surface.DrawRect(0, 0, w, h)
 	end
 
-	baitsbutton:SetPos(3 + (xpx - 6) / 2, 24) -- baits shop start of conf
-	baitsbutton:SetSize((xpx - 6) / 2, 22)
+	baitsbutton:SetPos(3 + (xpx - 6) / 3, 24) -- baits shop start of conf

```

Comment:
Not sure if this was part of the original code but when you do operations with "magic numbers" please make them constants such as CONSTANT = 3 it greatly helps reading